<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TraderLoginForm</class>
 <widget class="QWidget" name="TraderLoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Times New Roman&quot;;
background-color: rgb(50, 50, 50);
color: rgb(220, 220, 220);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,2,1,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 60);</string>
     </property>
     <property name="text">
      <string>Создание нового кассира</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="l1">
       <property name="text">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_trader_name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 55, 55);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="l2">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_trader_password">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 55, 55);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="l_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_save_trader">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Добавть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
